p4_constraints/backend/errors.h

```diff
@@ -1,3 +1,21 @@
+/*
```

Comment:
Do we know why the scrip is using mutltline comments here? 
Of course it's just a nit, but it would be nice to be consistent. My preference would be to use single-line comments consistently.

Reply:
I think that addlicense is using the file name suffix to decide what style of comments to add, and this is what it does for .h files.  See https://github.com/google/addlicense/blob/master/main.go#L292-L293